packages/ui/components/input-datepicker/test/lion-input-datepicker.test.js

```diff
@@ -495,11 +495,11 @@ describe('<lion-input-datepicker>', () => {
       const el = await fixture(html`<lion-input-datepicker></lion-input-datepicker>`);
```

Comment:
I am surprised to see that datepicker is a modal dialog, especially when it is presented as local next to the invoker.
On small screens (when it is a bottomsheet), this should be fine, but on desktop size screens it should be a local overlay.